lib/puma/configuration.rb

```diff
@@ -135,6 +135,7 @@ class Configuration
       environment: 'development'.freeze,
       # Number of seconds to wait until we get the first data for the request.
       first_data_timeout: 30,
+      force_shutdown_after: -1,
```

Comment:
This is the default but it lived in the method definition. IMO it should live here because it's a configurable option.